templates/docs/examples/patterns/navigation/deprecated-subnav.html

```diff
@@ -88,4 +88,7 @@
 <script>
   {% include 'docs/examples/patterns/navigation/_script.js' %}
 </script>
+<script>
```

Comment:
That's great, thanks.

One little detail, I think it would be good to put it into single `<script>` block (both the include and the line that runs the `initNavDropdowns`. It's because in the embedded examples in code snippet we extract single script tag only. So the contents of the include are visible, but not the init line.

We could probably update the example script to fix that, but merging the scripts into one will be a simple workaround ;)

Reply:
Ahhhhh, very good spot - updating